src/app/api/actions/supprimerUneNoteDeContexteAction.test.ts

```diff
@@ -0,0 +1,39 @@
+import * as nextCache from 'next/cache'
+
+import { supprimerUneNoteDeContexteAction } from './supprimerUneNoteDeContexteAction'
+import * as ssoGateway from '@/gateways/NextAuthAuthentificationGateway'
+import { SupprimerUneNoteDeContexte } from '@/use-cases/commands/SupprimerUneNoteDeContexte'
+
+describe('supprimer une note de contexte', () => {
+  it('quand une note de contexte est supprimée, alors cela renvoie un succès et le cache de la page appelante est purgé', async () => {
+    // GIVEN
+    vi.spyOn(ssoGateway, 'getSessionSub').mockResolvedValueOnce('userFooId')
+    vi.spyOn(nextCache, 'revalidatePath').mockImplementationOnce(vi.fn())
+    vi.spyOn(SupprimerUneNoteDeContexte.prototype, 'handle').mockResolvedValueOnce('OK')
+
+    // WHEN
+    const messages = await supprimerUneNoteDeContexteAction({
+      path: '/gouvernance/11',
+      uidGouvernance: 'uidGouvernance',
+    })
+
+    // THEN
+    expect(messages).toStrictEqual(['OK'])
+    expect(nextCache.revalidatePath).toHaveBeenCalledWith('/gouvernance/11')
+    expect(SupprimerUneNoteDeContexte.prototype.handle).toHaveBeenCalledWith({
+      uidEditeur: 'userFooId',
+      uidGouvernance: 'uidGouvernance',
```

Comment:
```suggestion
      uidGouvernance: 'gouvernanceFooId',
```
Pour garder de la cohérence